src/pages/version/tasks/PatchTasksTable.tsx

```diff
@@ -15,15 +15,17 @@ import { queryString } from "utils";
 const { toSortString } = queryString;
 
 interface Props {
+  isPatch: boolean;
   tasks: VersionTasksQuery["version"]["tasks"]["data"];
   sorts: SortOrder[];
   loading: boolean;
 }
 
```

Comment:
(placement of comment is random)

This might be an opportunity to update base commit / previous commit wording for task metadata! We do this on the patch metadata but not on the task metadata

<img width="304" alt="Screenshot 2023-04-27 at 4 38 03 PM" src="https://user-images.githubusercontent.com/47064971/234985240-b0d1d45a-952f-4845-af01-fc7ee3ffea89.png">

Reply:
Oh yeah, I noticed that the base commit hash is wrong but I think it actually does link to the correct previous version. I think we'd just need to get `previousVersion` from the `versionMetadata` and display its revision, but evaluating it in a ticket works too.